src/plugins/blocksmith/components/article-block-modal/CustomizeOptions.jsx

```diff
@@ -6,6 +6,8 @@ import {
   FormGroup,
   Input,
   Label,
+  Icon,
+  UncontrolledTooltip,
```

Comment:
@nanetteCodes use `@triniti/cms/plugins/common/components/uncontrolled-tooltip` it has some custom timing that we will probably eventually move into admin-ui. same for the others